Ginger/GingerCore/ALM/Qtest/ExportToQtest.cs

```diff
@@ -0,0 +1,594 @@
+                        testParams.AddParam(var.Name.ToLower(), "String");
+                        testParams.Save();
+                        test.Post();
+                    }
+                    catch (Exception ex) { Reporter.ToLog(eLogLevel.ERROR, $"Method - {MethodBase.GetCurrentMethod().Name}, Error - {ex.Message}", ex); }
+                }
+            }
+            description = description.Replace("<<&Parameters&>>", paramsSigns);
+
+            string actsDesc = string.Empty;
+            if (activity.Acts.Count > 0)
+            {
+                actsDesc = "Actions:<br />";
+                foreach (Act act in activity.Acts)
+                    actsDesc += act.Description + "<br />";
+            }
+            description = description.Replace("<<&Actions&>>", actsDesc);
+            step.StepDescription = description;
+
+            string expectedTemplate =
+               "<html><body><div align=\"left\"><font face=\"Arial\"><span style=\"font-size:8pt\"><<&Expected&&>></span></font></div></body></html>";
+            step.StepExpectedResult = expectedTemplate.Replace("<<&Expected&&>>", activity.Expected);
+
+            step.Post();
+
+            activity.ExternalID = step.ID.ToString();
+            return true;
+        }
+
+        public static bool ExportBusinessFlowToQtest(BusinessFlow businessFlow, TestSet mappedTestSet, string uploadPath, ObservableList<ExternalItemFieldBase> testSetFields, ref string result)
+        {
+            TestSet testSet;
+            ObservableList<ActivitiesGroup> existingActivitiesGroups = new ObservableList<ActivitiesGroup>();
+            try
+            {
+                if (mappedTestSet == null)
+                {
+                    //##create new Test Set in QC
+                    TestSetFactory TestSetF = (TestSetFactory)mTDConn.TestSetFactory;
+                    testSet = (TestSet)TestSetF.AddItem(System.DBNull.Value);                    
+
+                    //set the upload path
+                    TestSetTreeManager treeM = (TestSetTreeManager)mTDConn.TestSetTreeManager;
+                    ISysTreeNode testSetParentFolder = (ISysTreeNode)treeM.get_NodeByPath(uploadPath);
+                    testSet.TestSetFolder = testSetParentFolder.NodeID;
+                }
+                else
+                {
+                    //##update existing test set
+                    //testSet = mappedTestSet;
+                    testSet = ImportFromQC.GetQCTestSet(mappedTestSet.ID.ToString());
+                     
+                    TSTestFactory testsF = (TSTestFactory)testSet.TSTestFactory;
+                    List tsTestsList = testsF.NewList("");
+                    foreach (TSTest tsTest in tsTestsList)
+                    {
+                        ActivitiesGroup ag = businessFlow.ActivitiesGroups.Where(x => (x.ExternalID == tsTest.TestId.ToString() && x.ExternalID2 == tsTest.ID.ToString())).FirstOrDefault();
+                        if (ag == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=5395796)